go/vt/sqlparser/comments.go

```diff
@@ -583,42 +562,43 @@ func checkDirective(stmt Statement, key string) bool {
 	return false
 }
 
-// GetPriorityFromStatement gets the priority from the provided Statement, using DirectivePriority
-func GetPriorityFromStatement(statement Statement) (string, error) {
-	commentedStatement, ok := statement.(Commented)
-	// This would mean that the statement lacks comments, so we can't obtain the workload from it. Hence default to
-	// empty priority
+type QueryHints struct {
```

Comment:
I like this struct, but it's missing a few query hints. Don't we want all query hints here?

Reply:
It is only containing the ones being set on the session. Trying to keep the changes to specific areas.

Reply:
maybe call it SessionQueryHints to make it clear that it does not contain all hints?